web-common/src/proto/gen/rill/runtime/v1/queries_pb.ts

```diff
@@ -3758,16 +3975,22 @@ export class ColumnTimeRangeResponse extends Message<ColumnTimeRangeResponse> {
  */
 export class TimeRangeSummary extends Message<TimeRangeSummary> {
   /**
+   * Not null
+   *
    * @generated from field: google.protobuf.Timestamp min = 1;
    */
   min?: Timestamp;
 
   /**
+   * Not null
+   *
    * @generated from field: google.protobuf.Timestamp max = 2;
    */
   max?: Timestamp;
 
   /**
+   * Not null
```

Comment:
I think "not null" in a lot of these cases is a little confusing. If the type is `interval?: TimeRangeSummary_interval`, what i need to know is whether or not it's optional, not whether it's null or not -- null is not mentioned anywhere

Reply:
I meant `not optional` by saying `not null`, should I rename it?

Reply:
We can have 3 cases:
```
{
  "a": "string",
  "field": object
}
```
```
{
  "a": "string",
  "field": null
}
```
and 
```
{
 "a": "string"
}
```
`Not null` for `field` says last 2 cases aren't possible. `Not optional` doesn't negates the 2nd one though.

Reply:
I think the way I'd recommend clarifying this distinction is by specifically calling out when a field may be null. When combined with optional/required, that could give us 4 cases I suppose. 

(case 1) "Required, Not null" or "Not optional, Not null"-- We know the field must be present; ("Required" is probably a bit better, I think that is more standard.)
(case 2) "Required, Nullable" -- We  know that the field must be present, but it may be null
(case 3) "Optional, Not Null"

(maybe case 4) "Optional, Nullable" 

I don't think there is any reason to try to collapse both of those orthogonal concepts down into a single term though, it's better to be full explicit I think. Thanks @egor-ryashin !